packages/shorebird_cli/lib/src/commands/patch/ios_patcher.dart

```diff
@@ -52,6 +52,30 @@ class IosPatcher extends Patcher {
 
   String get _appDillCopyPath => p.join(buildDirectory.path, 'app.dill');
 
+  /// The value of --split-debug-info-path if specified.
+  String? get splitDebugInfoPath =>
+      argResults[CommonArguments.splitDebugInfoArg.name] as String?;
+
+  // TODO(felangel): make this dynamic based on the platform and arch.
+  /// The name of the split debug info file.
+  static const splitDebugInfoFileName = 'app.ios-arm64.symbols';
```

Comment:
This is duplicated with the releaser?

Reply:
It's duplicated between ios_patcher and ios_framework_patcher but we can refactor in a future PR if we want to share more of the logic.